code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -1072,7 +1080,7 @@
 	. = TRUE
 	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
 		return
-	new /obj/item/mounted/frame/alarm_frame(get_turf(user))
+	new /obj/item/mounted/frame/alarm_frame(loc)
```

Comment:
Is there a reason to change this?

Reply:
Are the frames on the wall, or on the turf the mob is on? I can't check on SDMM right now.

Reply:
yeah the frames are on the wall, so you get the funny thing where you get the stuff dropping below your feet instead of below the object, which is weird. Shoulda mentioned that in the PR description

Reply:
The alarms themselve should be on the turf the mob is on and pixel shifted, not the wall. Otherwise they can be seen and messed with from the other side.

Reply:
These are the items and not the ones on them walls (discussed on discord)